include/zephyr/usb_c/usbc.h

```diff
@@ -294,6 +337,82 @@ void usbc_set_policy_cb_get_rdo(const struct device *dev,
  */
 void usbc_set_policy_cb_is_snk_at_default(const struct device *dev,
 					  const policy_cb_is_snk_at_default_t cb);
+
+/**
+ * @brief Set the callback used to get the Rp value that should be placed on
+ *	  the CC lines
+ *
+ * @param dev Runtime device structure
+ * @param cb callback
+ */
+void usbc_set_policy_cb_get_src_rp(const struct device *dev,
+				   const policy_cb_get_src_rp_t cb);
```

Comment:
General comment regarding this expansion of the API.  Would it make sense to group these policy callbacks by function? Something like:
```
void usbc_set_policy_cb();  /* Shared callbacks needed for src and snk */
void usbc_set_policy_cb_snk(); /* snk only callbacks */
void usbc_set_policy_cb_src(); /* src only callbacks */
```

Reply:
Will make change in a separate PR.